timestamp-impl/src/main/java/com/palantir/timestamp/AvailableTimestamps.java

```diff
@@ -65,25 +90,29 @@ public synchronized void fastForwardTo(long newMinimum) {
         upperLimit.increaseToAtLeast(newMinimum + ALLOCATION_BUFFER_SIZE);
     }
 
-    private long lastHandedOut() {
+    private synchronized long lastHandedOut() {
```

Comment:
note to reviewers: technically this was already safe, but want `synchronized` for documentation and avoids having to litter with `assert Thread.holdsLock(this);` that bloats bytecode more.